src/en/alternativescans/build.gradle

```
@@ -0,0 +1,10 @@
+ext {
+    extName = "Alternative Scans"
+    extClass = ".AlternativeScans"
+    themePkg = "madara"
+    baseUrl = "https://alternativescans.icu"
+    extVersionCode = 2
+    overrideVersionCode = 1
+}
+
+apply from: "$rootDir/common.gradle"
```

```suggestion
ext {
    extName = "Alternative Scans"
    extClass = ".AlternativeScans"
    extVersionCode = 1
}

apply from: "$rootDir/common.gradle"
```
themePkg is only needed for multisrc sources